### PR TITLE
[Register] Create register screen

### LINKED ISSUE #28: [Register] Create register screen
## Overview (Required)

- Create screen that regisiter to dodam

## Todo

- [x] Create info screen
- [x] Create sign page
- [x] Create signup business logic

### PR DESCRIPTION
**Pull request by wsi1212**
## Overview (Required)

-  Create register screen

## Video

https://github.com/Team-B1ND/dodamdodam-android/assets/105199741/600614d5-acfc-4dd6-a3bd-b846c9c09b77

## ETC

- Performing login feature server error handling
- Network.core.model.Response data change to nullable

## Issue

- close #28

### CODE REVIEW

File: designsystem/src/main/java/com/b1nd/dodam/designsystem/util/PhoneVisualTransformation.kt [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+                    maskIndex = nextDigitIndex
+                }
+                append(trimmed[textIndex++])
+                maskIndex++
+            }
+        }
+
+        return TransformedText(annotatedString, PhoneOffsetMapper(mask, maskNumber))
+    }
+    override fun equals(other: Any?): Boolean {
```
Comment (Mercen-Lee) [inline designsystem/src/main/java/com/b1nd/dodam/designsystem/util/PhoneVisualTransformation.kt:34]: Space missing between functions

**Comment by Mercen-Lee** [comment]
PR Title is wrong

regisiter -> register

**Review by mercen-lee — CHANGES_REQUESTED** [review]
Change required

**Review by mercen-lee — APPROVED** [review]
LGTM 👍

**Comment by qrsang** [comment]
LGBT 🏳️‍🌈

**Pull request merged**